### ISSUE TITLE
Package Hl7.DSTU2.FhirPath v.1.9.1.0 - something wrong with the signature

### ISSUE THREAD

**Issue by BaronRastignak** [FirelyTeam/firely-net-sdk#1955]
Hello,
We upgraded packages Hl7.Fhir.DSTU2 from 1.4.0 to 1.9.1 and Hl7.Fhir.R4 from 1.4.0 to 3.8.0 in our solutions, and now IIS-hosted ASP.NET application crashes on start (solution was built without errors).

`System.IO.FileLoadException: Could not load file or assembly "Hl7.DSTU2.FhirPath, Version=1.9.1.0, Culture=neutral, PublicKeyToken=d706911480550fc3" or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)`

Output from sn.exe for Hl7.DSTU2.FhirPath:
```
> sn -v %USERPROFILE%\.nuget\packages\hl7.dstu2.fhirpath\1.9.1\lib\net45\Hl7.DSTU2.FhirPath.dll
%USERPROFILE%\.nuget\packages\hl7.dstu2.fhirpath\1.9.1\lib\net45\Hl7.DSTU2.FhirPath.dll is a delay-signed or test-signed assembly
```

```
> sn -v %USERPROFILE%\.nuget\packages\hl7.dstu2.fhirpath\1.9.1\lib\netstandard2.0\Hl7.DSTU2.FhirPath.dll
%USERPROFILE%\.nuget\packages\hl7.dstu2.fhirpath\1.9.1\lib\netstandard2.0\Hl7.DSTU2.FhirPath.dll is a delay-signed or test-signed assembly
```

Error is not reproducible in desktop environment, probably due to strong-name validation bypass.

**Comment by marcovisserFurore** [comment]
Hi @BaronRastignak,

Thanks for your feedback. 

And you're right about the signing. This has slipped through our quality control.

We sign the dll's with this pattern:
```
    $(Build.SourcesDirectory)\src\Hl7.Fhir*\bin\Release\*\Hl7.Fhir*.dll -Exclude Hl7.Fhir*Tests
```
and
```
    $(Build.SourcesDirectory)\common\src\Hl7.Fhir*\bin\Release\*\Hl7.Fhir*.dll -Exclude Hl7.Fhir*Tests
```

And this special FhirPath library, named Hl7.DSTU2.FhirPath.dll is not included in the file pattern...

We will fix this with probably a new version, because we cannot alter the current 1.9.1 version on NuGet

**Comment by marcovisserFurore** [comment]
I released an hotfix for 1.9.1, see release https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v1.9.1.1-dstu2.

The new version is **1.9.1.1**.

Thanks again for reporting this.
